src/zip.c

```diff
@@ -48,6 +49,9 @@
         }                      \
     } while (0)
 
+#include "miniz.h"
```

Comment:
I'm curious - why did you move includes from the top, here?

Reply:
so that some declarations declared earlier were available for these files, for example

`#define _CRT_SECURE_NO_WARNINGS`

it was an experiment, I forgot to put it back.